tests/cli/test_cli_organize.py

```
@@ -42,6 +52,42 @@ def _mock_result(
+                Panel.fit(
+                    "[bold yellow]First-time setup required[/bold yellow]\n\n"
+                    "File Organizer needs to be configured before use.\n"
+                    "Run the setup wizard to get started:\n\n"
+                    "  [bold cyan]file-organizer setup[/bold cyan]\n\n"
+                    "This will detect your system capabilities and configure\n"
+                    "the optimal AI models for your hardware.",
+                    border_style="yellow",
+                )
+            )
+            console.print()
+            raise typer.Exit(code=1)
```

Ruff (0.15.7)

[error] Undefined name `typer`

(F821)